packages/viem/src/utils.ts

```diff
@@ -0,0 +1,5 @@
+export function jsonStringifyBigInt(value: unknown) {
```

Comment:
can we leave this in `index.ts` for now? and tiiiny nit for a renaming: `jsonStringifyWithBigInt`